libs/d3d12/main.c

```diff
@@ -206,3 +206,13 @@ HRESULT WINAPI DLLEXPORT D3D12GetInterface(REFCLSID rcslid, REFIID iid, void **d
         return E_NOINTERFACE;
     return IVKD3DCoreInterface_GetInterface(core, rcslid, iid, debug);
 }
+
+void WINAPI DLLEXPORT D3D12PIXEventsReplaceBlock()
```

Comment:
At least the return types should be correct here and return E_NOTIMPL. I doubt these functions return void.

Reply:
Similarly, printing a FIXME probably screws with some registers so all these should probably do is return.... something.

Reply:
Yes. I approve. I've updated the codes to return the E_NOTIMPL. Could you help review them?